torchdynamo/allowed_functions.py

```diff
@@ -82,15 +82,13 @@ def _disallowed_function_ids():
         torch.cuda.amp.autocast_mode.autocast,
         torch.distributions.constraints.is_dependent,
         torch.distributions.normal.Normal,
-        torch.get_rng_state,
         torch.inference_mode,
         torch.set_anomaly_enabled,
         torch.set_autocast_cache_enabled,
         torch.set_autocast_cpu_dtype,
         torch.set_autocast_cpu_enabled,
         torch.set_autocast_enabled,
         torch.set_autocast_gpu_dtype,
-        torch.set_rng_state,
```

Comment:
Why remove these?

Reply:
Before this PR these were getting inlined, which seems incorrect.   The change below makes them not inline anymore.

Reply:
Nice :)